src/Language/PureScript/Errors.hs

```diff
@@ -911,8 +911,13 @@ prettyPrintSingleError (PPEOptions codeColor full level showDocs relPath) e = fl
       paras [ line "An exhaustivity check was abandoned due to too many possible cases."
             , line "You may want to decompose your data types into smaller types."
             ]
-    renderSimpleErrorMessage (UnusedImport name) =
-      line $ "The import of module " <> markCode (runModuleName name) <> " is redundant"
+
+    renderSimpleErrorMessage (UnusedImport mn qualifier) =
+      let
+        mark = markCode . runModuleName
+        qual' q = " (qualified as " <> mark q <> ")"
+        qual = maybe "" qual'
+      in line $ "The import of module " <> mark mn <> qual qualifier <> " is redundant"
```

Comment:
Bit of a nit but may I suggest copying GHC and saying "The qualified import of [module] as [qualified-name] is redundant"?

Reply:
Sure, that seem likes a good idea. I'll change the message, and I'll probably be able to push up the changes by tonight.

(By the way, please, pick as many nits as you want. I appreciate any tips/advice you guys have for me.)